spec/template.dd

```diff
@@ -391,9 +391,9 @@ immutable(S)
 )
 
     $(P This is especially useful when used with inheritance. For example,
-        you might want to implement a final base method which returns a derived
-        class type. Typically you would return a base type, but this won't allow
-        you to call or access derived properties of the type:)
+        consider implementing a final base method which returns a derived
```

Comment:
Given the context of the sentence, "consider implementing" is a bit confusing. It sounds like a recommendation to implement something. That put me in the wrong frame for the rest of the sentence and I had to shift gears. "consider the implementation of" sounds more like you're asking the reader to ponder a hypothetical (which is what you're doing).